src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -127,6 +127,13 @@ public async Task<HttpResponseMessage> SendAsync(CancellationToken cancellationT
 
             try
             {
+                // Works around linker issue where it tries to eliminate QuicStreamAbortedException
+                // https://github.com/dotnet/runtime/issues/57010
+                #if TARGET_MOBILE
```

Comment:
Should this be `TARGET_MOBILE || TARGET_BROWSER`?

Reply:
This file isn't compiled on Browser:

https://github.com/dotnet/runtime/blob/2b3f76c8e3ccb28aefbf5ca879a7cf5933b2642a/src/libraries/System.Net.Http/src/System.Net.Http.csproj#L142-L173